abuild.in

```diff
@@ -1511,7 +1512,11 @@ update_abuildrepo_index() {
 	done
 	subpkg_unset
 
-	[ -z "$DESCRIPTION" ] && DESCRIPTION="$repo $(cd $startdir && git describe || true)"
+	if [ -z "$DESCRIPTION" ]; then
+		[ -z "$git_path" ] && DESCRIPTION="$repo"
+		[ -n "$git_path" ] && DESCRIPTION="$repo $(cd $startdir && git describe)"
```

Comment:
This is quite confusing, please use normal `if… else … fi` for exclusive code paths like this.

Reply:
Now that you say it, it is a lot simpler with normal `if... else...fi` indeed! I've updated it.